app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -0,0 +1,70 @@
+module EmsMiddlewareHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name type hostname port)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    items = []
+    items.concat(%i(middleware_servers middleware_deployments))
+    items
+  end
+
+  def textual_group_status
+    %i(refresh_status)
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+  def textual_refresh_status
+    last_refresh_status = @ems.last_refresh_status.titleize
+    if @ems.last_refresh_date
+      last_refresh_date = time_ago_in_words(@ems.last_refresh_date.in_time_zone(Time.zone)).titleize
+      last_refresh_status << " - #{last_refresh_date} Ago"
+    end
+    {
+      :label => _('Last Refresh'),
+      :value => [{:value => last_refresh_status},
+                 {:value => @ems.last_refresh_error.try(:truncate, 120)}],
+      :title => @ems.last_refresh_error
+    }
+  end
+
+  def textual_group_topology
+    items = %w(topology)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
```

Comment:
This doesn't seem to follow the pattern of the other textual_group methods... @abonas?